tools/cli/src/index.ts

```diff
@@ -219,6 +219,20 @@ create
     handleResponse(result);
   });
 
+create
+  .command('label')
+  .description('Create a label')
```

Comment:
Add descriptions for command parameters (just like you did with 'remove')

Reply:
done